internal/restconfig/restconfig.go

```diff
@@ -547,3 +583,39 @@ func ConfigureTestFramework(args []string) error {
 
 	return nil
 }
+
+var submarinerWarned = false
```

Comment:
This means that if we process multiple contexts, we’ll only get a warning for the first one :-/